app/views/stages/registration_identifier_stage/show.html.erb

```diff
@@ -3,6 +3,17 @@
     <div class="row">
       <article class="col">
         <h1 class="govuk-heading-xl">Your registration identifier</h1>
+
+        <p class="govuk-body">We need to record your Spacecraft Registration Identifier (SRI).</p>
```

Comment:
Ah, we do refer to an `SRI` here, so maybe disregard my previous comment.

Reply:
I'm updating this heading to "Your registration ID" (see above)